### PR TITLE
[Catalog] Improve Ink demo color contrast

### PR DESCRIPTION
**Pull request by romoore**
The Ink example's color contrast ratios were too low to pass WCAG 2.1 AA-level
contrast requirements. Specifically the blue-on-white (Ink ripple effect) and
dark grey on light grey (text) were too low.  Switching to using the MDC
Palettes instead of custom colors.

**Before**
ink-contrast-before

**After**
ink-contrast-after

Related to #4506

### CODE REVIEW

**Comment by romoore** [comment]
@yarneo @codeman7 Ready for review

File: components/Buttons/src/MDCButton.m [hunk 1]
```diff
@@ -261,6 +261,7 @@ - (void)layoutSubviews {
 }
 
 - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent *)event {
+  NSLog(@"Button (%p) touch area: %@", self, NSStringFromCGRect(UIEdgeInsetsInsetRect(CGRectStandardize(self.frame), self.hitAreaInsets)));
```
Comment (yarneo) [inline components/Buttons/src/MDCButton.m]: please remove logging

Comment (romoore) [inline components/Buttons/src/MDCButton.m]: Oops, reverted!

**Review by yarneo — APPROVED** [review]

**Review by codeman7 — APPROVED** [review]

**Comment by jverkoey** [comment]
Is this able to merge?

**Pull request merged**